azuredeploy.json

```diff
@@ -605,6 +612,27 @@
                 }
             }
         },
+        {
+            "condition": "[not(bool(equals(parameters('searchType'), 'none')))]",
```

Comment:
Sorry it didn't occur to me earlier, but I think installing a Tika VM should be switchable as well. That is, customer should be able to choose the Elastic Search or the Azure Search, but also be able to choose not to have an additional Tika VM (service). It should be doable by adding another switch (yet another...) to the main template and have it and-ed here.

Reply:
@hosungsmsft - i am still working on getting tika service optional in the template, will notify you once this has been done.